openfisca_web_api/scripts/serve.py

```diff
@@ -48,7 +48,7 @@ def read_user_configuration(default_configuration, command_line_parser):
     configuration = default_configuration
     args, unknown_args = command_line_parser.parse_known_args()
 
-    if args.configuration_file:
+    if 'configuration_file' in vars(args) and args.configuration_file:
```

Comment:
Now that we throw a clear error message in case of a bare `openfisca`, I think we don't need this anymore.

```suggestion
    if args.configuration_file:
```